presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -778,6 +794,11 @@ Procedures
     Unregisters given, existing partition in the metastore for the specified table.
     The partition data is not deleted.
 
+HDFS Erasure Coding
```

Comment:
Let's remove the section header and move to the *HDFS Configuration* section. However, I'm not sure it's useful to document this at all, as this seems like a low level detail of HDFS that people would just expect to work (like encryption or failover).